CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 
 ### Dependency updates
 
+## [18.2.2]
+
+### Fixed
+
+- `Toast.tsx`: Break long words to prevent text overflow ([@JorenSaeyTL](https://github.com/JorenSaeyTL)) in ([#2492](https://github.com/teamleadercrm/ui/pull/2492))
```

Comment:
```suggestion
- `Toast`: Break long words to prevent text overflow ([@JorenSaeyTL](https://github.com/JorenSaeyTL)) in ([#2492](https://github.com/teamleadercrm/ui/pull/2492))
```